planet/cli/orders.py

```diff
@@ -263,6 +263,12 @@ async def create(ctx, request: str, pretty):
     type=types.JSON(),
     help="""Credentials for cloud storage provider to enable cloud delivery of
     data. Can be a json string, filename, or '-' for stdin.""")
+@click.option(
+    '--no-stac',
```

Comment:
I don't use click a lot, so others might have a more informed comment on this (cc @sgillies), but it could be nice to have `--stac/--no-stac` be the pair of options here (with the default being `False` (e.g. https://click-docs-cn.readthedocs.io/zh_CN/latest/options.html#boolean-flags).  That would let everybody be explicit if they wanted to, and both `--stac` and `--no-stac` options would continue to work if we ever change the default.

Reply:
@tschaub @sgillies I'm not sure I see the purpose of `--stac/--no-stac`, if we're going to provide STAC by default - i.e., what's the purpose of having a `--stac` flag?

Reply:
After our discussion, I the `--stac/--no-stac` boolean pair flags are helpful for being explicit. See: https://github.com/planetlabs/planet-client-python/pull/676/commits/affba48d659055fe102fe088278246479ac1b6c6